build.gradle.kts

```diff
@@ -45,7 +54,110 @@ subprojects {
     }
 }
 
-repositories {
-    mavenLocal()
-    mavenCentral()
+subprojects {
+    if (name == "exposed-bom") return@subprojects
+
+    apply(plugin = "org.jetbrains.kotlin.jvm")
+
+    testDb("h2") {
+        withContainer = false
+        dialects("H2", "H2_MYSQL", "H2_PSQL", "H2_MARIADB", "H2_ORACLE", "H2_SQLSERVER")
+
+        dependencies {
+            dependency("com.h2database:h2:${Versions.h2_v2}")
+        }
+    }
+
+    testDb("h2_v1") {
+        withContainer = false
+        dialects("H2", "H2_MYSQL")
+
+        dependencies {
+            dependency("com.h2database:h2:${Versions.h2}")
+        }
+    }
+
+    testDb("sqlite") {
+        withContainer = false
+        dialects("sqlite")
+
+        dependencies {
+            dependency("org.xerial:sqlite-jdbc:${Versions.sqlLite3}")
+        }
+    }
+
+    testDb("mysql") {
+        port = 3001
+        dialects("mysql")
+        dependencies {
+            dependency("mysql:mysql-connector-java:${Versions.mysql51}")
+        }
+    }
+
+    testDb("mysql8") {
+        port = 3002
+        dialects("mysql")
+        dependencies {
+            dependency("mysql:mysql-connector-java:${Versions.mysql80}")
+        }
+    }
+
+    testDb("mariadb") {
+        port = 3000
+        dialects("mariadb")
+        dependencies {
+            dependency("org.mariadb.jdbc:mariadb-java-client:${Versions.mariaDB_v2}")
+        }
+    }
```

Comment:
This seems to be an extra task for MariaDB version 2 that duplicates the task `mariadb_v2` below.
I just confirmed on TC that tests are being run 3 times for MariaDB instead of just 2 times.

Reply:
thanks, will fix